modules/files.nix

```diff
@@ -210,17 +210,14 @@ in
       '') (filter (v: v.onChange != "") (attrValues cfg))
     );
 
-    home-files = pkgs.stdenv.mkDerivation {
-      name = "home-manager-files";
-
-      nativeBuildInputs = [ pkgs.xlibs.lndir ];
-
-      preferLocalBuild = true;
-      allowSubstitutes = false;
-
-      # Symlink directories and files that have the right execute bit.
-      # Copy files that need their execute bit changed.
-      buildCommand = ''
+    # Symlink directories and files that have the right execute bit.
+    # Copy files that need their execute bit changed.
+    home-files = pkgs.runCommand
+      "home-manager-files"
+      {
+        nativeBuildInputs = [ pkgs.xlibs.lndir ];
```

Comment:
You're missing `preferLocalBuild = true;`. It's not set [by default](https://github.com/NixOS/nixpkgs/blob/88c3183581c1ff9af6ab020352aa12bdeffc9602/pkgs/build-support/trivial-builders.nix#L5-L9).

Reply:
My bad. I got confused and mixed it up with `writeText`. I'll fix it later. Thanks for pointing it out! 🙂